ssl/quic/quic_wire.c

```diff
@@ -47,14 +47,14 @@ int ossl_quic_wire_encode_frame_ack(WPACKET *pkt,
                                            : OSSL_QUIC_FRAME_TYPE_ACK_WITHOUT_ECN;
 
     uint64_t largest_ackd, first_ack_range, ack_delay_enc;
-    size_t i, num_ack_ranges = ack->num_ack_ranges;
+    uint64_t i, num_ack_ranges = ack->num_ack_ranges;
     OSSL_TIME delay;
```

Comment:
Very pedantic, but I guess since `ack->num_ack_ranges` should be `size_t` we should have a check here to make sure it does not exceed `UINT64_MAX` (though a platform where `SIZE_MAX` > `UINT64_MAX` is hard to imagine...)

You can ignore me on this if you like though.,

Reply:
I agree that this might be overly pedantic.  Ignored :wink: